include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t              maxDescriptorSetUpdateAfterBindInlineUniformBlocks;
+    //uint32_t              maxInlineUniformTotalSize;
+    //VkBool32              integerDotProduct8BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct8BitSignedAccelerated;
+    //VkBool32              integerDotProduct8BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct16BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct16BitSignedAccelerated;
+    //VkBool32              integerDotProduct16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct32BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct32BitSignedAccelerated;
+    //VkBool32              integerDotProduct32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct64BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct64BitSignedAccelerated;
+    //VkBool32              integerDotProduct64BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated;
+    //VkDeviceSize          storageTexelBufferOffsetAlignmentBytes;
+    //VkBool32              storageTexelBufferOffsetSingleTexelAlignment;
+    //VkDeviceSize          uniformTexelBufferOffsetAlignmentBytes;
+    //VkBool32              uniformTexelBufferOffsetSingleTexelAlignment;
```

Comment:
TODO

Reply:
let's keep only 
```cpp
 storageTexelBufferOffsetAlignmentBytes
 uniformTexelBufferOffsetAlignmentBytes
 ```
 
 and always enable the feature in Vk 1.3 or the extension when its present.
 
 ### Spoofing on Vk<1.3 when VK_EXT_texel_buffer_alignment is not present
 
 set
 ```cpp
 storageTexelBufferOffsetAlignmentBytes = vk10_props.minTexelBufferOffsetAlignment;
 uniformTexelBufferOffsetAlignmentBytes = vk10_props.minTexelBufferOffsetAlignment;
 ```
 
 ### GL

treat GL as a VK 1.1 impl with no extension and treat `GL_TEXTURE_BUFFER_OFFSET_ALIGNMENT` as if its `vk10_props.minTexelBufferOffsetAlignment`